packages/search-ui/src/__tests__/actions/setFilter.test.js

```diff
@@ -64,54 +66,63 @@ describe("#setFilter", () => {
 
   it("Adds a new filter and removes old filters", () => {
     expect(
-      subject("test", "value2", undefined, {
+      subject("test", "value2", undefined, undefined, {
         initialFilters: [
-          { field: "initial", values: ["value"], type: "all" },
-          { field: "test", values: ["value1"], type: "all" }
+          {
+            field: "initial",
+            values: ["value"],
+            type: "all",
+            outerType: "all"
+          },
+          { field: "test", values: ["value1"], type: "all", outerType: "all" }
         ]
       }).filters
     ).toEqual([
-      { field: "initial", values: ["value"], type: "all" },
-      { field: "test", values: ["value2"], type: "all" }
+      { field: "initial", values: ["value"], type: "all", outerType: "all" },
+      { field: "test", values: ["value2"], type: "all", outerType: "all" }
     ]);
   });
 
   it("Adds an 'any' type filter", () => {
-    expect(subject("test", "value", "any").filters).toEqual([
-      { field: "test", values: ["value"], type: "any" }
+    expect(subject("test", "value", "any", "any").filters).toEqual([
```

Comment:
I have the same comment for this entire file, it'd be better to add new tests for outerType, I think.